.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- [ ] Your project is missing `.hintrc` and `.stylelintrc.json` files in the root directory. Kindly add these linter configuration files. You can find these files in the [linter template from microverse here](https://github.com/microverseinc/linters-config/tree/master/html-css).